selfservice/flow/profile/extension.go

```diff
@@ -0,0 +1,74 @@
+package profile
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+
+	"github.com/pkg/errors"
+	"github.com/tidwall/gjson"
+
+	"github.com/ory/kratos/selfservice/form"
+
+	"github.com/ory/jsonschema/v3"
+	"github.com/ory/x/jsonschemax"
+)
+
+const (
+	extensionName = "disableIdentifiersExtension"
+	extensionKey  = "ory.sh/kratos"
+)
+
+type DisableIdentifiersExtension struct {
+	disabled bool
+}
+
+func RegisterNewDisableIdentifiersExtension(compiler *jsonschema.Compiler) {
+	if _, ok := compiler.Extensions[extensionName]; ok {
```

Comment:
I don't think we really care. There is no up/downside of overwriting the extension so we can reduce code a bit :)